hal/tpm_io_linux.c

```diff
@@ -68,28 +68,34 @@
         #define TPM2_I2C_HZ   400000 /* 400kHz */
     #else
         /* SPI */
-        #ifdef WOLFTPM_MICROCHIP
-            /* Microchip ATTPM20 uses CE0 */
-            #define TPM2_SPI_DEV_CS "0"
-        #elif defined(WOLFTPM_ST33)
-            /* STM ST33HTPH SPI uses CE0 */
-            #define TPM2_SPI_DEV_CS "0"
-        #elif defined(WOLFTPM_NUVOTON)
-            /* Nuvoton NPCT75x uses CE0 */
-            #define TPM2_SPI_DEV_CS "0"
-        #else
-            /* OPTIGA SLB9670/SLB9762 and LetsTrust TPM use CE1 */
-            #define TPM2_SPI_DEV_CS "1"
+        #ifndef TPM2_SPI_DEV_CS
+            #ifdef WOLFTPM_MICROCHIP
+                /* Microchip ATTPM20 uses CE0 */
+                #define TPM2_SPI_DEV_CS "0"
+            #elif defined(WOLFTPM_ST33)
+                /* STM ST33HTPH SPI uses CE0 */
+                #define TPM2_SPI_DEV_CS "0"
+            #elif defined(WOLFTPM_NUVOTON)
+                /* Nuvoton NPCT75x uses CE0 */
+                #define TPM2_SPI_DEV_CS "0"
+            #else
+                /* OPTIGA SLB9670/SLB9762 and LetsTrust TPM use CE1 */
+                #define TPM2_SPI_DEV_CS "0"
```

Comment:
Why change this? The comment still says CE1, but it uses CE0. This case is when we know it is an Infineon, this is not the auto-detect case.

Reply:
Since the autodetect will be updated to always be 0 to 4, there's no need for this change either.  I'll revert.